cmd/csi/livenessprobe/cmd.go

```diff
@@ -0,0 +1,44 @@
+package livenessprobe
+
+import (
+	dtcsi "github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi/livenessprobe"
+	"github.com/Dynatrace/dynatrace-operator/pkg/logd"
+	"github.com/Dynatrace/dynatrace-operator/pkg/version"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const use = "csi-livenessprobe"
+
+var probeTimeout, csiAddress, healthPort string
+
+func New() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  use,
+		RunE: run(),
+	}
+
+	addFlags(cmd)
+
+	return cmd
+}
+
+func addFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&probeTimeout, "probe-timeout", "9s", "probe timeout")
```

Comment:
why string and not time.Second, and why do we want to hardcode it here and not inside helm?